### PR TITLE
Milestone 4: Comments & likes

### PR DESCRIPTION
**Pull request by Anna-Myzukina**
Create models with associations and implement all requested features for comments and likes.

### CODE REVIEW

File: app/models/like.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Like < ApplicationRecord
```
Comment (mcrd25) [inline app/models/like.rb]: WE are missing some associations here.
Hint: Associations similar to those of comments

File: app/views/devise/registrations/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+  <%= image_tag 'facebook1.png' %>
+</div>
+<div class="col-md-2">
+</div>
+  <div class="col-md-5">
+  <h2 class="pl">Create a New Account</h2>
+  <h4 class="pl">It`s quick and easy</h4>
+  <div class="form-row">
+    <div class="field">
+    <%= f.label :username%><br/>
```
Comment (mcrd25) [inline app/views/devise/registrations/new.html.erb]: [OPTIONAL]
Maybe this should be the First Name label 😛

Comment (mcrd25) [reply app/views/devise/registrations/new.html.erb]: 😅

File: app/views/devise/registrations/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    <%= f.text_field :username, autofocus: true, autocomplete: "name", placeholder: "First name" %>
+    </div>
+
+      <div class="field">
+      <%= f.label :sirname%><br/>
```
Comment (mcrd25) [inline app/views/devise/registrations/new.html.erb]: [OPTIONAL]
Surname is misspelled (Please Change)

Comment (mcrd25) [reply app/views/devise/registrations/new.html.erb]: [OPTIONAL]
You can also simply put Last Name.

File: app/views/posts/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+<div class="container">
+<div class="row">
+<p id="notice"><%= notice %></p>
+
+<div class="col-md-4">
+<% @posts.each do |post| %>
+<p>Username:<%= post.user.username %></p> 
+<p>Email:<%= post.user.email %></p> 
+<% end %>
+<button class='btn btn-primary'><%= link_to 'New Post', new_post_path %></button>
```
Comment (maelfosso) [inline app/views/posts/index.html.erb]: You can't put a `<a />` link into a button. 
I had to change it like this 
```ruby
<%= link_to 'New Post', new_post_path, class: 'btn btn-primary' %>
```
to make it work

File: app/views/posts/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+<h2><%= current_user.username %></h2> 
+</div>
+<div class="under-buner-menu">
+<a href="#">Lorem</a>
+<a href="#">Ipsum</a>
+<a href="#">Set</a>
+<a href="#">Loram</a>
+</div>
+</div>
+
```
Comment (maelfosso) [inline app/views/posts/index.html.erb]: Please, indent your code to make it easily readable

File: app/views/posts/index.html.erb [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+<%= render "comment/form" ,post: post=%> 
+<% post.comments.each do |comment| %>
+<%= render 'comment/comment', comment: comment,post: post =%>
+<% end %>
+</div>
+<div>
+<td><%= render partial: "likes", locals: {post: post}%></td> 
+<td><i class="fa fa-thumbs-up"></i><%= post.likes.count %> <%= (post.likes.count) == 1 ? 'Like' : 'Likes'%></td><hr>
+<span class='comment-count'>Comments: <%= post.comments.count =%> </span>
+</div>
```
Comment (maelfosso) [inline app/views/posts/index.html.erb]: Please, move that part up.
Right after the post body, will be ok.

**Comment by Elukoye** [comment]
Completed milestone four - comments and likes section together with RSpec tests, 
kindly review.
Thanks

**Comment by samgaco** [comment]
Hi @Anna-Myzukina and @Elukoye :high_brightness: 

All of the required steps are corrected. You can move on :+1: 

------------------------------

* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

**Pull request merged**